sig-architecture/api-review-process.md

```diff
@@ -0,0 +1,257 @@
+
+* The feedback should be made in the issue with APPROVED or  REVIEWED (for externally-maintained CRDs or external components where there is only feedback, not approval)
+
+* Final status in the issue should be given along with @ notifications for the submitter(s) so they are informed when the review is complete
+
+If **rejected**:
+
+* If completely rejected, e.g. "please do this work outside the Kubernetes org" - an explanation of why the change was rejected - appeals can be requested from the api-approvers mailing list ([kubernetes-api-reviewers@googlegroups.com](mailto:kubernetes-api-reviewers@googlegroups.com)) where the moderator will coordinate a follow-up review.  If that request results in another rejection, there is no further appeal.
+
+* If rejected with "changes requested" - an explanation in the review document of what exactly needs to be changed and why (prior decisions, standards, etc.)
+
+* All applicable in-tree or critical PRs and Issues will be noted as not approved with UNAPPROVED until they successfully pass a review
+
+* Final status in the API Review issue should be given along with @ notifications for the submitter(s) so they are informed when the review is complete
+
+## The Moderator Role
+
+The moderator role is staffed by SIG Architecture and manages the API review backlog on behalf of the reviewer team. They will ensure that reviews are finished within a reasonable time, that information is correct, and that appropriate state labels are applied. They may also help prioritize the backlog, or move cards across the project board. Their mission is to help reviewers spend the majority of their efforts on performing reviews, not doing process administrivia. They may also work with the review team to schedule face-to-face review sessions as needed, or ensure the review is added to the SIG-Architecture meeting agenda.
+
+## Process mechanics (see diagram above)
+
+Timing of API review requests matters.  The larger the change the more time that must be afforded.  New API resources (aka Kinds) may require significantly more thought than single field additions.  API reviews that are requested too late in a release cycle may not complete in time to make the release.  Plan ahead. Also, if you are changing an approved API, you must consult with the [kubernetes-api-reviewers@googlegroups.com](mailto:kubernetes-api-reviewers@googlegroups.com) list to ensure it is still consistent with the approvals already granted. From a process perspective, you would request a new review in that case.
+
+New APIs (groups or Kinds) or substantial changes require KEPs.  Major changes without KEPs will be rejected.
+
+1. Create an API Review request issue in [https://github.com/kubernetes-sigs/architecture-tracking](https://github.com/kubernetes-sigs/architecture-tracking) - the work will be tracked in the corresponding project board
```

Comment:
might be helpful to link to the board. https://github.com/kubernetes-sigs/architecture-tracking/projects/3, right?